hushline/__init__.py

```diff
@@ -140,6 +142,10 @@ def inject_variables() -> dict[str, Any]:
             directory_verified_tab_enabled=app.config["DIRECTORY_VERIFIED_TAB_ENABLED"],
             is_onion_service=request.host.lower().endswith(".onion"),
             is_premium_enabled=bool(app.config.get("STRIPE_SECRET_KEY", False)),
+            registration_enabled=data.get(OrganizationSetting.REGISTRATION_ENABLED, True),
```

Comment:
For security, we might want to make this false by default, no?

Reply:
The problem is registration needs to be enabled by default to register the first user on the personal server.

A solution would be to set REGISTRATION_ENABLED to False by default, but in practice it's enabled until there is at least 1 user. Thoughts @brassy-endomorph ?